src/Microsoft.OData.Edm/Csdl/CsdlJsonWriterSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="CsdlJsonWriterSettings.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+
+namespace Microsoft.OData.Edm.Csdl
+{
+    /// <summary>
+    /// The options/setting for CSDL writer
+    /// </summary>
+    public class CsdlJsonWriterSettings
+    {
+        internal static readonly CsdlJsonWriterSettings Default = new CsdlJsonWriterSettings();
+
+        /// <summary>
+        /// Gets/sets a value indicating whether the writer write large integers as strings.
+        /// The IEEE754Compatible=true parameter indicates that the service MUST serialize Edm.Int64 and Edm.Decimal numbers as strings.
+        /// This is in conformance with [RFC7493]. If not specified, or specified as IEEE754Compatible=false, all numbers MUST be serialized as JSON numbers.
+        /// </summary>
+        public bool IsIeee754Compatible { get; set; }
+
+        /// <summary>
+        /// Serialize enum member expression value as numeric or symblic value.
+        /// true as numeric value as: "17"
+        /// False as symblic value as "Red, Striped"
+        /// </summary>
+        public bool EnumMemberExpressionAsNumeric { get; set; }

```

Comment:
I would like to remove this option and always serialize as symbolic (unless there is no matching symbolic name, then as numeric value).

Reply:
I think we can give customers more options to choose, make the lib more easy to use.
By default, it's "false" and output as symbolic. 

For advanced developer, he can change this if he wants to get the numeric. 

It's open for more discussion.

---
In reply to: [429386392](https://github.com/OData/odata.net/pull/1775#discussion_r429386392) [](ancestors = 429386392)

Reply:
The protocol says to serialize as symbolic number (where available), so not sure why we would offer this option?

We can always add it later, but I'd like to remove it for now (since once we ship it we can't really take it back).


---
In reply to: [429473648](https://github.com/OData/odata.net/pull/1775#discussion_r429473648) [](ancestors = 429473648,429386392)

Reply:
removed.

---
In reply to: [440480550](https://github.com/OData/odata.net/pull/1775#discussion_r440480550) [](ancestors = 440480550,429473648,429386392)

Reply:
thanks.

---
In reply to: [444559565](https://github.com/OData/odata.net/pull/1775#discussion_r444559565) [](ancestors = 444559565,440480550,429473648,429386392)